### PR TITLE
[AIRFLOW-9] Improving docs to meet Apache's standards

### PR DESCRIPTION
**Pull request by mistercrunch**
Following the guidelines here:
http://incubator.apache.org/guides/sites.html

Related:
https://issues.apache.org/jira/browse/INFRA-12028

### CODE REVIEW

File: docs/project.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+----------
+
+- @mistercrunch (Maxime "Max" Beauchemin)
+- @r39132 (Siddharth "Sid" Anand)
+- @criccomini (Chris Riccomini)
+- @bolkedebruin (Bolke de Bruin)
+- @artwr (Arthur Wiedmer)
+- @jlowin (Jeremiah Lowin)
+- @patrickleotardif (Patrick Leo Tardif)
+- @aoen (Dan Davydov)
```
Comment (criccomini) [inline docs/project.rst:25]: Think we lost our latest committer @syvineckruyk in this list

**Comment by criccomini** [comment]
LGTM after Steve is added. :)

**Comment by jlowin** [comment]
Funny, I still can't look at names in quotes without thinking of ["The Yellow Dart"](https://www.youtube.com/watch?v=TNCRwxwMhjg)... I can't believe that video is 13 years old.

**Comment by mistercrunch** [comment]
Docs as of this branch are now here:
http://airflow.incubator.apache.org/

**Pull request merged**